site/content/release-notes/3.4.4.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "3.4.4"
+---
+RDF4J 3.4.4 is a patch release fixing 4 bugs including a bug with how transactions with RDF* were handled by the MemoryStore.
```

Comment:
The milestone actually has 5 issues in it (though arguably one of them is not a bug): https://github.com/eclipse/rdf4j/milestone/61?closed=1